### DISCUSSION TITLE
Anyone got a resource pack already made for the public?

### DISCUSSION THREAD

**Discussion by BlankZQz** [TartaricAcid/TouhouLittleMaid#370] [General]
Hi, yes, I would love to use others' creations! Uh, yeah, that's it. Thank you!

**Comment by TartaricAcid** [comment]
1.12.2 publicly released model packs can be downloaded directly in the game
1.16.5 doesn't have this feature yet, but it will be done soon

**Reply by BlankZQz** [reply]
Thank you for the reply!
